public/projects.html

```diff
@@ -0,0 +1,61 @@
+<html>
+  <head>
+    <title>Projects</title>
+  </head>
+
+  <header>
+    <nav>
+      <a href="/">Home Page</a>
+      <a href="about">About Page</a>
+      <a href="projects">Projects</a>
+      <a href="blog">Blog</a>
+    </nav>
+  </header>
+
+  <h1>PROJECTS</h1>
+
+  <body>
+    <p>
+      Below are some of the projects I have been working on.
+    </p>
+    <ul>
+
+      <li>
+        <a href="https://github.com/shannachau/C3Projects--Calculator/tree/svc/master" target="_blank">Calculator Project</a>
+      </li>
+        <ul>
```

Comment:
This nesting is a little awkward. Nesting lists is totally a thing; we use them to do things like have multi-level navigation or create sub-lists of related items. However, to create a semantic nested list, you would want the sub-list as a child of the `<li>` item, like this:

```html
<ul>
  <li>
    I'm the list item
    <ul>
      <li> And I'm the sub-list item</li>
    </ul>
  </li>
</ul>
```

Notice that the nested `<ul>` appears inside the list-item (before the closing tag). This indicates a hierarchal relationship.